### ISSUE TITLE
Show/hide map parts

### ISSUE THREAD

**Issue by jan-gerard** [OpenOrienteering/mapper#627]
Map parts are very useful. But there is no way to hide certain map parts. I think that would be very useful, for example when I imported an OpenStreetMap and start drawing my own map on top, sometimes I want to see the OSM objects, and sometimes I only want my own map. 
I suggest to add an option to show/hide map parts.

**Comment by Symbian9** [comment]
Did you mean that map parts  need similar window as for template?

**Comment by jan-gerard** [comment]
I don't think that custom transparency values would be needed. Just on and off.

**Comment by dg0yt** [comment]
I considered this occassionally. The parts could be grouped under the "- Map -" line in the template window.

But if you just want to draw on top, open the .osm file as a template.

**Comment by jan-gerard** [comment]
That's a good idea. I didn't know .osm files could be templates. 
Still, sometimes I would like to hide certain parts of a map.

**Comment by dg0yt** [comment]
Everything which can be a map can also be a template. But not the other way :-)
